tests/unit/unit-util.cpp

```diff
@@ -47,6 +47,28 @@ class UnitUtilTests : public ::testing::Test,
   TermVec symbols;
 };
 
+class UnitUtilDimacsTests
+    : public ::testing::Test,
+      public ::testing::WithParamInterface<SolverConfiguration>
+{
+ protected:
+  void SetUp() override
+  {
+    SolverConfiguration se = GetParam();
+    SolverConfiguration sc(se.solver_enum, true);
+    s = create_solver(sc);
+
+    boolsort = s->make_sort(BOOL);
+    for (size_t i = 0; i < 30; ++i)
+    {
+      symbols.push_back(s->make_symbol("x" + std::to_string(i), boolsort));
+    }
+  }
+  SmtSolver s;
+  Sort boolsort;
+  TermVec symbols;
```

Comment:
What are these for?

Reply:
I removed TermVec, if I don't mention the other two the compiler throws an error of "undeclared variable"

Reply:
yes, because you are using them! the google tests framework makes these class fields available in the test bodies.